OpenRA.Mods.Common/Activities/Air/Land.cs

```diff
@@ -44,7 +44,7 @@ public override Activity Tick(Actor self)
 				return NextActivity;
 			}
 
-			Fly.FlyToward(self, plane, d.Yaw.Facing, WDist.Zero);
+			Fly.FlyToward(self, plane, d.Yaw.Facing, plane.Info.LandAltitude);
```

Comment:
This won't work if you want planes to be able to land both at multiple places with different altitudes.  Wouldn't it be better to use `target.Z` here?